dipy/testing/decorators.py

```diff
@@ -52,9 +53,15 @@ def doctest_skip_parser(func):
 # xvfbwrapper) conditioned on an environment variable (that we set in
 # .travis.yml for these cases):
 use_xvfb = os.environ.get('TEST_WITH_XVFB', False)
+if use_xvfb == 'true':
```

Comment:
Ah no. I am wrong. I see that in the travis file it says https://github.com/nipy/dipy/blob/master/.travis.yml#L50